week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,9 @@
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var sum = 15 + 8;
+var percentageOfStudents = (15 / 23) * 100;
+var percentageOfMentors = (8 / 23) * 100;
+var roughPercentageOfStudents = Math.round(percentageOfStudents);
+var roughPercentageOfMentors = Math.round(percentageOfMentors);
+console.log("Percentage students:" + roughPercentageOfStudents + "%");
+console.log("Percentage mentors:" + roughPercentageOfMentors + "%");
```

Comment:
For this exercise, the solution should have created one function that calculates the percentage, and one function that creates a message. Try rewriting this solution to use these two functions.

Reply:
Tanks for your feedback ,  i  will correct my mistakes on a few exercises.